### DISCUSSION TITLE
Initial pose estimates for ns-process-data

### DISCUSSION THREAD

**Discussion by alfonsomp13** [nerfstudio-project/nerfstudio#3536] [Q&A]
Hi, 
I’m working with a dataset that includes camera images and pose estimates derived from GNSS data (horizontal heading and assumed planar movement). I’ve stored these initial poses in a transforms.json file, and I was wondering if I can use this file with the ns-process-data command. Specifically, if I place the transforms.json with the images in the same folder, will COLMAP (through NeRFStudio) automatically read and refine these initial poses during the preprocessing step? 

If this isn’t supported yet, can you suggest alternative methods to utilize these initial pose estimates and speed up the preprocessing?

Thanks!

**Comment by ncan33** [comment]
I really need to do this as well. I have some initial pose estimates that have around 5 to 15% error. I am wondering if I can feed this as initial conditions to `ns-process-data`
